lib/syntax_tree/visitor/mutation_visitor.rb

```diff
@@ -322,14 +322,6 @@ def visit_else(node)
         )
       end
 
-      # Visit a Elsif node.
-      def visit_elsif(node)
-        node.copy(
-          statements: visit(node.statements),
-          consequent: visit(node.consequent)
-        )
-      end
-
```

Comment:
This one also doesn't seem to have any tests. The only difference between this method and `visit_if` is that `visit_if` copies the `predicate`:

```
      def visit_if(node)
        node.copy(
          predicate: visit(node.predicate),
          statements: visit(node.statements),
          consequent: visit(node.consequent)
        )
      end
```

It seems like that should also be done for `elsif`, but maybe it shouldn't be for some reason?

Reply:
Nah it should, I think it was missed when I templated out that method.